### ISSUE TITLE
Revise Descriptions When OSCAL Content Returned to Client

### ISSUE THREAD

**Issue by brian-ruf-ezd** [EasyDynamics/oscal-rest#90]
### Description

As a developer issuing OSCAL REST API calls to retrieve OSCAL content, I need to understand how to request the OSCAL format I require. 

### Acceptance Criteria

- [x] For each model, the description for the following endpoints is updated as defined below
  - [x] `GET /[model-name]/{Identifier}`
  - [x] `GET /[model-name]/{Identifier}/snapshot/{Identifier}`
  - [x] `GET /profile/{Identifier}/resolved-catalog`
  - [x] `GET /profile/{Identifier}/resolved-snapshot/{Identifier}`

### Description Content

The current description should be maintained for each of the four endpoints, with the following _added_ to that content:

```
When processing this request, the implementation _must_ honor the `Accept` HTTP header for JSON, XML, and YAML. When `Accept` is set to:
- `application/json` or `application/oscal+json` return the OSCAL content in JSON format;
- `application/xml` (preferred by RFC 7303) or `text/xml` (deprecated by RFC 7303) or `application/oscal+xml` return the OSCAL content in XML format; or
- `application/yaml` or `text/yaml` return the OSCAL content in YAML format.

When returning the OSCAL content, the implementation _must_ set the `Content-type` header to:
- `application/json` when returning the OSCAL content in JSON format;
- `application/xml` when returning the OSCAL content in XML format; or
- `application/yaml` when return the OSCAL content in YAML format.

```

**Comment by brian-ruf-ezd** [comment]
@mpemy changes look good!

**Comment by brian-ruf-ezd** [comment]
Full addressed in PR #72. Closing.
